backend.native/tests/external/codegen/box/constants/kt9532.kt

```diff
@@ -22,7 +20,10 @@ object B {
 fun box(): String {
     if (A.a !== B.a) return "Fail 1: A.a !== B.a"
 
-    if (A.b !== B.b) return "Fail 2: A.b !== B.b"
+    if (A.b !== B.b) {
+        println("${A.b} & ${B.b}")
```

Comment:
println?

Reply:
This part should be removed.  Was added accidentally when I was investigating the reason of failure of this test